### DISCUSSION TITLE
How to use clear/no color?

### DISCUSSION THREAD

**Discussion by FancyFurret** [tui-cs/Terminal.Gui#1569] [Q&A]
Hello! I would like to remove all background color from my app, how can I accomplish this? Ive tried using `Colors.Base.Normal = Application.Driver.MakeAttribute(Color.White, Color.Black);` but it doesn't look like there is any Color.None/Color.Clear/etc.

My terminal emulator has a nice blurred background that I still want to see when running my gui.cs app, I don't want it to be filled with black/blue/etc., it doesn't look very nice.

Thanks! :)

**Comment by tznind** [comment]
Hey, thanks for enquiring.  What Terminal are you using? on which OS? I don't think this is something that is currently supported but it looks like it is possible at least with the ncurses driver (https://github.com/andmarti1424/sc-im/issues/164 and https://stackoverflow.com/questions/2403399/ncurses-transparent-console-background).

**Reply by FancyFurret** [reply]
I’m using hyper on macOS

**Reply by tznind** [reply]
So on my linux laptop with Terminology shell and an animated gif background I am able to get the effect you are looking for with:
 
```csharp
_baseColorScheme.Normal = new Terminal.Gui.Attribute(-1);
```
But -1 is transparent Foreground AND Background.  Presumably there is some magic number that will be transparent background with x foreground color but I don't think it is mapped.

Also I think this isn't going to work for the Windows driver either.  There -1 seems to give me a whitish grey color.  

![shot-2022-01-06_14-36-15](https://user-images.githubusercontent.com/31306100/148399471-40f64c2e-a481-4e6a-bb82-29ac466aa498.jpg)
